.github/pull_request_template.md

```diff
@@ -0,0 +1,34 @@
+## Description
+
+Thank you for your contribution to the PySyft repository.
+Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context.
+List any dependencies that are required for this change.
```

Comment:
"New dependencies" could probably be a checkbox in the `Type of change` list below.

Reply:
I guess it isn't required. Removed it.